__mocks__/react-native-localize.js

```diff
@@ -0,0 +1,36 @@
+const getLocales = () => [
+  { countryCode: 'US', languageTag: 'en-US', languageCode: 'en', isRTL: false },
+  { countryCode: 'EC', languageTag: 'es-EC', languageCode: 'es', isRTL: false },
+];
+
+const findBestAvailableLanguage = () => ({ languageTag: 'es', isRTL: 'false' });
+const getNumberFormatSettings = () => ({
+  decimalSeparator: '.',
+  groupingSeparator: ',',
+});
+
+const getCalendar = () => 'gregorian';
+const getCountry = () => 'ES';
+const getCurrencies = () => ['USD'];
+const getTemperatureUnit = () => 'celsius';
+const getTimeZone = () => 'Ecuador/Guayaquil';
+const uses24HourClock = () => true;
+const usesMetricSystem = () => true;
+
+const addEventListener = jest.fn();
+const removeEventListener = jest.fn();
+
+export {
```

Comment:
For consistency purposes, it would be better to add the inline `export` statement on the function definitions, like this:
```js
export const findBestAvailableLanguage = () => ({ languageTag: 'es', isRTL: 'false' });
```

Other than that, this looks great!

Reply:
Done 👍